demos/index.html

```diff
@@ -518,101 +518,52 @@ <h3>
         Phase 2: Accessible Actions / Input Events
       </h3>
       <p>
-        Listen to input events directly from assistive technology.
+        To preserve the privacy of assistive technology users, 
```

Comment:
Is it worth putting a link to the discussion on this topic, seeing as it got such a strong reaction? I'm also not sure I understand the use of the word typically in this case. Like, is it typically because that's the current norm? Because it only does it under certain circumstances?

Reply:
Link added. 

Not sure where "typically" came from to be honest; reworded to "... cause a synthesised DOM event to be triggered where appropriate".